.github/workflows/llm_integration.yml

```diff
@@ -341,17 +341,6 @@ jobs:
           python3 llm/client.py trtllm qwen-7b
           rm -rf docker_env
           docker rm -f $(docker ps -aq)
-      - name: flan-t5-xxl pre-compiled model with python backend
-        working-directory: tests/integration
-        run: |
-          rm -rf models
-          echo -en "CUDA_VISIBLE_DEVICES=0,1,2,3" > docker_env
-          python3 llm/prepare.py trtllm flan-t5-xxl
```

Comment:
could we also remove the test in the python files?

Reply:
Spared those because if we added support in the next week, it would be easier to add. Never mind, will remove those too.

Reply:
Done removed those too